pac4j-saml-config-db/src/test/java/org/pac4j/saml/dbclient/dao/impl/CachingSamlClientDaoImplTest.java

```diff
@@ -0,0 +1,167 @@
+/*
+  Copyright 2012 - 2015 pac4j organization
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+ */
+package org.pac4j.saml.dbclient.dao.impl;
+
+import static org.junit.Assert.*;
+import static org.mockito.Matchers.*;
+import static org.mockito.Mockito.*;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.pac4j.saml.dbclient.dao.api.DbLoadedSamlClientConfigurationDto;
+import org.pac4j.saml.dbclient.dao.api.SamlClientDao;
+
+
+/**
+ * Unit test of {@link CachingSamlClientDaoImpl}.
+ * 
+ * @author jkacer
+ */
+public class CachingSamlClientDaoImplTest {
```

Comment:
Tests classes end with `Tests` in pac4j, you may or may not follow this convention on your own.